src/remoteConfig.ts

```diff
@@ -22,6 +23,10 @@ type RemoteConfigValue = {
   enableBalance: boolean;
   approvedStoreKitSandboxUsers: string[];
   coreProductId: string;
+  coresRoleRules: {
+    regions: string[];
+    role: CoresRole;
+  }[];
```

Comment:
We can specify multiple rules to block which regions are restricted and how, first found rule applies.